### DISCUSSION TITLE
tooltip 无法消失

### DISCUSSION THREAD

**Discussion by sanheart** [ant-design/ant-design#35156] [Q&A]
antd: 4.19.5
react: 16.8.6

使用tooltip时，hover移除后
`<div class="ant-tooltip ant-tooltip-placement-top  zoom-big-fast-leave zoom-big-fast-leave-active zoom-big-fast" style="pointer-events: none; left: 584px; top: 211px; transform-origin: 50% 74.3125px;"><div class="ant-tooltip-content"><div class="ant-tooltip-arrow"><span class="ant-tooltip-arrow-content"></span></div><div class="ant-tooltip-inner" role="tooltip">/static/fumidash/polarise_logo_light.png</div></div></div>`

但页面上仍存在此tootip

**Comment by andrew-tacklit** [comment]
I'm facing the same issue, any solution to this?
